do not override compiler and do not strip

The default makefile sets the compiler to g++ or gcc. This leads to a wrong architecture when cross-compiling.
Remove the hardcoded compiler and just append the flags to CXX and CC.

Upstream-Status: Pending
Signed-off-by: Raphael Freudiger <raphael.freudiger@siemens.com>
Reviewed-By: Pascal Bach <pascal.bach@siemens.com>

diff -Nurp p7zip_15.14.1_orig/makefile.machine p7zip_15.14.1/makefile.machine
--- p7zip_15.14.1_orig/makefile.machine	2016-03-23 20:37:47.000000000 +0100
+++ p7zip_15.14.1/makefile.machine	2016-06-17 15:33:39.720454477 +0200
@@ -2,7 +2,7 @@
 # makefile for Linux (x86, PPC, alpha ...)
 #
 
-OPTFLAGS=-O -s
+OPTFLAGS=-O
 
 ALLFLAGS=${OPTFLAGS} -pipe \
     	-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE \
@@ -10,8 +10,6 @@ ALLFLAGS=${OPTFLAGS} -pipe \
 	-D_7ZIP_LARGE_PAGES \
 	$(LOCAL_FLAGS)
 
-CXX=g++
-CC=gcc
 CC_SHARED=-fPIC
 LINK_SHARED=-fPIC -shared
 
